core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -55,6 +55,8 @@ object ZkUtils {
   val LogDirEventNotificationPath = "/log_dir_event_notification"
   val KafkaAclPath = "/kafka-acl"
   val KafkaAclChangesPath = "/kafka-acl-changes"
+  val KafkaWildcardSuffixedAclPath = "/kafka-wildcard-acl"
```

Comment:
As per the mail thread, we should use 'Prefixed' rather than 'WildcardSuffixed' everywhere.

Reply:
I think we decided to use 'WildcardSuffixed' ;)

Reply:
Nope, the last suggestion on the thread was from @cmccabe and was for 'prefixed'. I'm also happy with this as its short!  `Wildcard Suffixed` is a bit of a mouthful and is less intuitive - a good test for this is to try and explain how "wildcard suffixed ACLs" interact with literal and wildcard acls.  It ends up being really confusing, as we've got 'wildcard-suffixed' and 'wildcard' ACLs - much easy to discuss and digest if its 'literal', 'prefixed' and 'wildcard' ACLs. :D

Reply:
Yes, I think "prefixed" makes sense.  Everyone who talks about this talks about it as "prefix" support.  it's even in the name of the KIP and the JIRA.